aws-rds-customdbengineversion/src/main/java/software/amazon/rds/customdbengineversion/Translator.java

```diff
@@ -27,6 +27,9 @@ public static CreateCustomDbEngineVersionRequest createCustomDbEngineVersionRequ
                 .description(model.getDescription())
                 .engine(model.getEngine())
                 .engineVersion(model.getEngineVersion())
+                .sourceCustomDbEngineVersionIdentifier(model.getSourceCustomDBEngineVersionIdentifier())
+                .useAwsProvidedLatestImage(model.getUseAwsProvidedLatestImage())
```

Comment:
If this parameter is not included in the describe response, you should designate it as a write-only parameter in the resource schema. Otherwise, it will consistently result in false drift.

However, if it is returned in the describe response, you should also include it in the `translateFromSdk` function.

Reply:
These are write-only / create-only attributes. I'll add them to lists. Thanks for the reminder!